.github/workflows/pull-request-ci.yaml

```diff
@@ -0,0 +1,319 @@
+name: CI
+on: push
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    env:
+      PINNED_TOOLS: true
+      TOOLS_DIR: ${{ github.workspace }}/tools
+    defaults:
+      run:
+        working-directory: kubecf
+    steps:
+    - name: Checkout KubeCF
```

Comment:
Can you please add the comment / space for all steps in all jobs?

Reply:
done